### ISSUE TITLE
multiple POST variables with same name only appear once

### ISSUE THREAD

**Issue by martinhaase** [UNINETT/SAML-tracer#29]
Doing a POST, looking at the variables for a checkbox with multiple values. Expecting:

> consent: read
> consent: write

However, SAML Tracer shows only one of them. I have not checked whether this affects also GET query parameters.

**Comment by khlr** [comment]
Hi Martin!

You're right, that's a bug (POST-only, GET works). 
My bad... I didn't correctly convert the function `parsePOST`.

For anyone interested: You can reproduce this using the following two JSFiddles (checking "coding" and "music"):

GET (https://jsfiddle.net/p7sfp61r/1/) produces:
```
GET
interest: coding 
interest: music 
other: 
```

POST (https://jsfiddle.net/p7sfp61r/) produces:
```
POST
interest: coding 
other: 
```

**Comment by khlr** [comment]
I just fixed it in pull request #30.

**Comment by martinhaase** [comment]
Thanks Jan for the quick response! Good stuff!
